crates/sparrow-compiler/src/dfg/const_eval.rs

```diff
@@ -34,7 +34,6 @@ pub(super) fn evaluate_constant(
             // now to fix various panics caused by not having *some* behavior defined.
             InstOp::CountIf => return Ok(ScalarValue::UInt32(Some(0))),
             InstOp::First => return Ok(inputs[0].null()),
-            InstOp::Lag => return Ok(inputs[0].null()),
```

Comment:
Makes sense for now. I added an item to the list of "deferred" things -- we probably want to either (a) implement `Lag` more directly or (b) do the thing we talked about where we use a "collect buffer" and support outputting just specific elements. Cloning everything to output the nested list just to take one element is definitely going to be inefficient.